.github/workflows/auto-bump.yml

```
@@ -30,22 +44,23 @@ jobs:
           echo "casks: $casks"
           echo "formulae=$formulae" >> $GITHUB_OUTPUT
           echo "casks=$casks" >> $GITHUB_OUTPUT
-      - uses: actions/create-github-app-token@v2
-        id: app-token
-        with:
-          app-id: ${{ secrets.APP_ID }}
-          private-key: ${{ secrets.APP_PRIVATE_KEY }}
       - uses: Homebrew/actions/bump-packages@main
         with:
           fork: false
           token: ${{ steps.app-token.outputs.token }}
           formulae: ${{ steps.enum-packages.outputs.formulae }}
           casks: ${{ steps.enum-packages.outputs.casks }}
+
+  merge-pr:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.actor == 'app/actionstokenprovider'
```

```suggestion
    if: github.event_name == 'pull_request' # TODO: && github.actor == 'app/actionstokenprovider'
```

Remove `github.actor` check for testing.